binding/web/src/rhino_worker.ts

```diff
@@ -9,16 +9,20 @@
   specific language governing permissions and limitations under the License.
 */
 
-import { fromBase64, fromPublicDirectory } from '@picovoice/web-utils';
+import {
+  fromBase64,
```

Comment:
do you still use `fromBase64` and `fromPublicDirectory` somewhere? ideally we shouldn't need them either here or in rhino itself